### PR TITLE
Replaces all internal white space with one space

### PR DESCRIPTION
**Pull request by adkron**
This is to remove tags like "first \n name" and make them "first name"

### CODE REVIEW

**Comment by adkron** [comment]
this was just supposed to be the top commit. Sorry, screwed up the pull request.

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,4 @@
 .bundle
 rdoc
 Gemfile.lock
-
+.rvmrc
```
Comment (ches) [inline .gitignore:4]: Please add `.rvmrc` to your global git `excludesfile`, it's a personal choice much like editors are.

**Comment by ches** [comment]
The change looks good, but the history is confusing -- could you rebase your topic branch on current `integration` branch please?

**Comment by adkron** [comment]
Sorry I screwed up the pull request. There are actually two things here because of that. If you want them both There is the replace all internal whitespace of tags with a single space, and allow a forced call to aggregate tags.

**Comment by adkron** [comment]
I did a rebase and fixed a conflict. When I added the file it disappeared and wouldn't let me go on. So I just merged and fixed the conflict by hand.

**Comment by adkron** [comment]
This should merge cleanly. I tried it locally. Sorry about all the confusion. I think I was working from and old branch.

**Pull request closed without merging**